mlir/include/mlir/Interfaces/ViewLikeInterface.h

```diff
@@ -113,64 +135,72 @@ inline void printDynamicIndexList(
     OpAsmPrinter &printer, Operation *op, OperandRange values,
     ArrayRef<int64_t> integers, TypeRange valueTypes = TypeRange(),
     AsmParser::Delimiter delimiter = AsmParser::Delimiter::Square) {
-  return printDynamicIndexList(printer, op, values, integers, {}, valueTypes,
-                               delimiter);
+  return printDynamicIndexList(printer, op, values, integers, /*scalables=*/{},
```

Comment:
Shall we rename `scalables` as `scalableFlags`?  The latter is more descriptive and, in general, makes more sense to me. Also, apologies for introducing `scalables` 😅